scripts/url-checker.sh

```diff
@@ -53,4 +53,4 @@ while read line; do
         problematic+="$status\t$line\n"
     fi
 done < <(echo "$uniqued")
-echo -e $problematic | column -t
+echo -e $problematic
```

Comment:
`column` is not preinstalled on the Namespace images. Removing this seems to produce the same output.